content/docs/concepts/what-is-istio/index.md

```diff
@@ -7,54 +7,91 @@ aliases:
     - /docs/concepts/what-is-istio/goals
 ---
 
-This document introduces Istio: an open platform to connect, manage, and secure microservices. Istio provides an easy way to create a network of deployed services with load balancing, service-to-service authentication, monitoring, and more, without requiring any changes in service code. You add Istio support to services by deploying a special sidecar proxy throughout your environment that intercepts all network communication between microservices, configured and managed using Istio's control plane functionality.
+Istio is an open platform to connect, manage, and secure microservices.
 
-Istio currently supports service deployment on Kubernetes, as well as services registered with Consul or Eureka and services running on individual VMs.
+Istio provides an easy way to create a network of deployed services with load
+balancing, service-to-service authentication, monitoring, and more. Istio
+achieves it without requiring any changes in service code. To add Istio
+support to services, you deploy a special sidecar proxy throughout your
+environment. The proxy intercepts all network communication between
+microservices. You can configure and manage the proxy using Istio's control
+plane functionality.
+
+Istio currently supports:
+
+* Service deployment on Kubernetes
+* Services registered with Consul or Eureka
+* Services running on individual virtual machines.
 
 ## Why use Istio?
 
-Istio addresses many of the challenges faced by developers and operators as monolithic applications transition towards a distributed microservice architecture. The term **service mesh** is often used to describe the network of
-microservices that make up such applications and the interactions between them. As a service mesh grows in size and complexity, it can become harder to understand
-and manage. Its requirements can include discovery, load balancing, failure recovery, metrics, and monitoring, and often more complex operational requirements
-such as A/B testing, canary releases, rate limiting, access control, and end-to-end authentication.
+Istio addresses the challenges developers and operators face as monolithic
+applications transition towards a distributed microservice architecture.
+
+The term **service mesh** is used to describe the network of microservices that
```

Comment:
Given that "service mesh" is two words, is it correct to call that a "term"?

Reply:
Yes, terms can be compound terms which are composed of more than one word.